content/en/docs/tasks/debug-application-cluster/crictl.md

```diff
@@ -41,13 +39,19 @@ of Kubernetes. Extract it and move it to a location on your system path, such as
 The `crictl` command has several subcommands and runtime flags. Use
 `crictl help` or `crictl <subcommand> help` for more details.
 
-`crictl` connects to `unix:///var/run/dockershim.sock` by default. For other
+`crictl` connects to `unix:///var/run/containerd.sock` by default. For other
```

Comment:
there is no default runtime, and "other runtimes", anymore. crictl will search a list of known runtimes, that's all

most likely you **don't** want to duplicate that list of endpoints here, but link to the crictl documentation instead ?

Reply:
I changed the wording to this and linked to the crictl docs for more information. 

You can set the endpoint for `crictl` by doing one of the following:

* Set the `--runtime-endpoint` and `--image-endpoint` flags.
* Set the `CONTAINER_RUNTIME_ENDPOINT` and `IMAGE_SERVICE_ENDPOINT` environment
  variables.
* Set the endpoint in the configuration file `--config=/etc/crictl.yaml`.

{{<note>}}
If you don't set an endpoint, `crictl` attempts to connect to a list of known
endpoints, which might result in an impact to performance.
{{</note>}}